robot/impl/local_robot.go

```diff
@@ -1065,6 +1065,24 @@ func (r *localRobot) discoverRobotInternals(query resource.DiscoveryQuery) (inte
 	}
 }
 
+func (r *localRobot) GetModelsFromModules(ctx context.Context) ([]*resource.ModuleModelDiscovery, error) {
```

Comment:
@cheukt @zaporter-work wasn't sure whether everything should live at the robot level or if I should add something to the module manager to better surface these messages.

Also not sure who I should be having to review this stuff, I assumed it would be cheuk but if theres a more appropriate person lmk

Reply:
kk moved to module manager. Just need to add tests for the server & manager and it should be ready